Corona-Warn-App/src/main/res/layout/event_registration_category_fragment.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
```

Comment:
Please don't use DataBinding Layout?

Reply:
Yes, when using `MaterialToolbar`, I don't need this.